modules/debts_api/lib/debts_api/v0/financial_status_report_service.rb

```diff
@@ -129,9 +133,9 @@ def submit_vha_fsr(form_submission)
         form_submission:,
```

Comment:
This here is the Sharepoint request right? Maybe we need a new metric like this:
```
request_start_time = Time.current
sharepoint_request.upload(
        form_contents: vha_form,
        form_submission
...
StatsD.measure("#{STATSD_KEY_PREFIX}.fsr.submit.sharepoint.latency", (Time.current - request_start_time) * 1000)
```

And then the one you have around the vbs request:

```
request_start_time = code that does start time
bs_response = vbs_request.post("#{vbs_settings.base_path}/UploadFSRJsonDocument",
StatsD code
                                      { jsonDocument: vha_form.to_json })
```